apps/wallet/src/ui/app/pages/accounts/ExportAccountPage.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { fromB64 } from '@mysten/sui.js/utils';
+import { bytesToHex } from '@noble/hashes/utils';
+import { useMutation } from '@tanstack/react-query';
+import { useMemo } from 'react';
+import { Navigate, useNavigate, useParams } from 'react-router-dom';
+
+import { HideShowDisplayBox } from '../../components/HideShowDisplayBox';
+import { VerifyPasswordModal } from '../../components/accounts/VerifyPasswordModal';
+import Alert from '../../components/alert';
+import Loading from '../../components/loading';
+import Overlay from '../../components/overlay';
+import { useAccounts } from '../../hooks/useAccounts';
+import { useBackgroundClient } from '_src/ui/app/hooks/useBackgroundClient';
+
+export function ExportAccountPage() {
+	const { accountID } = useParams();
+	const { data: allAccounts, isLoading } = useAccounts();
+	const account = useMemo(
+		() => allAccounts?.find(({ id }) => accountID === id) || null,
```

Comment:
is there a good reason for memo'ing this?

Reply:
To avoid the find loop I guess when not necessary. But yeah even without it, it wont make much difference